### DISCUSSION TITLE
Can't create order in ecommerce template

### DISCUSSION THREAD

**Discussion by turistua** [payloadcms/payload#5910] [Q&A]
Hi! 

I set up payload with ecommerce template, but when I try to place an order (either through admin panel, or through frontend), API
http://localhost:3000/api/orders stucks in pending state and no order is created.

Payload example:
`{"total":15,"orderedBy":1,"items":[{"id":"6620eaffed51005ab5db7a8d","price":15,"product":4,"quantity":1}]}`

I did no changes in the code, also I can add products and pages without such an issue. I use postgres as a DB.

Have anyone faced such an issue?

**Comment by paulpopus** [comment]
Looks like this issue may be out of date or with an older template, please let us know if this occurs with the latest template and plugin!

Closing this discussion in light of ecommerce plugin and template being released. We're going to capture discussions going forward in this RFC so we can better manage engagement on this topic: https://github.com/payloadcms/payload/discussions/14239

Thank you everyone!
